### PR TITLE
EVEREST-855 Namespaces validation improvements

### PR DESCRIPTION
**Pull request by oksana-grishchenko**
**Namespaces validation improvements**
---
**Problem:**
EVEREST-855

Make validation to cover more corner cases

**CHECKLIST**
---
**Jira**
- [ ] Is the Jira ticket created and referenced properly?

**Tests**
- [ ] Is an Integration test/test case added for the new feature/change?
- [ ] Are unit tests added where appropriate?

### CODE REVIEW

File: commands/install.go [hunk 1]
```diff
@@ -29,7 +29,13 @@ import (
...
 func newInstallCmd(l *zap.SugaredLogger) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "install",
+		Use: "install",
+		// The command expects no arguments. So to prevent users from misspelling and confusion
+		// in cases with unexpected spaces like
+		//       ./everestctl install --namespaces=aaa, a
+		// it will return
+		//        Error: unknown command "a" for "everestctl install"
+		Args:    cobra.NoArgs,
```
Comment (recharte) [inline commands/install.go:38]: This is a great idea :muscle:

File: pkg/kubernetes/client/client.go [hunk 2]
```diff
@@ -104,7 +104,7 @@ type Client struct {
 	customClientSet  *customresources.Client
 	apiextClientset  apiextv1clientset.Interface
 	dynamicClientset dynamic.Interface
-	dbClusterClient  *database.DBClusterClient
+	dbClusterClient  *database.DBClusterClient //nolint:structcheck,unused
```
Comment (recharte) [inline pkg/kubernetes/client/client.go:107]: Is this unused? Why do we keep it?

**Review by recharte — APPROVED** [review]

File: pkg/kubernetes/client/client.go (hunk 2 above)
Comment (oksana-grishchenko) [reply pkg/kubernetes/client/client.go:107]: I don't know how come, it's just the linter update highlighted that the field is not used. I wasn't sure if it was intended for some future plans, so decided not to change it. But on a second thought - I'll better remove it then
